build/configs/rtl8721csm/rtl8721csm_make_bin.sh

```diff
@@ -111,7 +111,10 @@ function concatenate_binary()
 		cat /root/tizenrt/build/tools/amebad/gnu_utility/km0_image2_all.bin /root/tizenrt/build/output/bin/km4_image2_all.bin  > /root/tizenrt/build/output/bin/km0_km4_image2.bin
 	else
 		echo "========== Concatenate_binary for TZ enabled =========="
-		cat /root/tizenrt/build/tools/amebad/gnu_utility/km0_image2_all.bin /root/tizenrt/build/output/bin/km4_image2_all.bin /root/tizenrt/build/tools/amebad/gnu_utility/km4_image3_all-en.bin /root/tizenrt/build/tools/amebad/gnu_utility/km4_image3_psram-en.bin > /root/tizenrt/build/output/bin/km0_km4_image2.bin
+		RDP_KEY="${CONFIG_AMEBAD_RDP_KEY:2}"
+		/root/tizenrt/build/tools/amebad/gnu_utility/imagetool.sh ${CONFIG_AMEBAD_RDP} ${RDP_KEY}
+
+		cat /root/tizenrt/build/tools/amebad/gnu_utility/km0_image2_all.bin /root/tizenrt/build/output/bin/km4_image2_all.bin /root/tizenrt/build/output/bin/km4_image3_all-en.bin /root/tizenrt/build/output/bin/km4_image3_psram-en.bin > /root/tizenrt/build/output/bin/km0_km4_image2.bin
 	fi
 }
```

Comment:
Is this working on each developer's env? Can you adapt the paths in the file to TOP_PATH macro, to make it more generic?

Reply:
Which one is having problem? /root/tizenrt/build/tools/amebad/gnu_utility/imagetool.sh?

Reply:
I dont know. I mean all the paths in this file are absolute and hardcoded, the better way would be to make path relative to your current directory. 

THIS_PATH=`test -d ${0%/*} && cd ${0%/*}; pwd`
TOP_PATH=${THIS_PATH}/../../../

Reply:
All path have been change.